src/entry-server.js

```diff
@@ -82,19 +82,16 @@ export default function initializeServer(createApp, serverOpts) {
                     }))
                     .then(() => resolve(app))
                     .catch((e) => {
-                        opts.logger.error('Error in middleware chain');
-                        opts.logger.error(e);
+                        opts.logger.log('Error in middleware chain', e);
```

Comment:
Combined these into one log statement and lowered to log.  This makes parsing SSR logs easier as it keeps the error alongside the message since messages are intermingled from all servers - and it also lowers to `log` since not every use-case here is an error.  For example, throwing a RedirectError to get out of the Vue SSR flow so we can redirect from express.  This allows the consuming application to choose its own log level.

Reply:
Also got rid of the fallback `Unknown error` errors since they never actually happen